doc/userguide/install.rst

```diff
@@ -77,38 +77,96 @@ Rust support::
 
   rustc, cargo
 
-  Not every distro provides Rust packages yet. Rust can also be installed
-  directly from the Rust project itself::
+  Rust packages can be found in package managers but some distros
+  don't provide or provide outdated Rust packages.
+  In case of insufficient version you can install Rust directly
+  from the Rust project itself::
 
-  https://www.rust-lang.org/en-US/install.html
+    1) Install Rust https://www.rust-lang.org/en-US/install.html
```

Comment:
Victor recently disagreed with using rustup for installing Rust (which is what is recommended in this link), is it ok, @victorjulien ?

Reply:
btw discussion is still ongoing https://github.com/OISF/suricata/pull/9612

Reply:
ok, thanks for the reference, will wait a bit before working on this again. 
But yes indeed it seems like apt install rust is the preferred way for the **user** docs

Tbh, I haven't really differentiated much user docs from the dev docs in terms of compilation but I can see the difference now.

Reply:
btw while reworking this I noticed package managers are still the preferred way mentioned in the paragraph above. I got confused by only seeing your comment and seeing the one line it applies to. 

For the copy-paste example, I can list rust, cargo, and cbindgen packages in individual distros below.